tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -799,6 +823,11 @@ public TopicCommandOptions(String[] args) {
                 "if set when creating topics, the action will only execute if the topic does not already exist.");
             excludeInternalTopicOpt = parser.accepts("exclude-internal",
                 "exclude internal topics when running list or describe command. The internal topics will be listed by default");
+            partitionSizeLimitPerResponseOpt = parser.accepts("partition-size-limit-per-response",
```

Comment:
In other cases where a new API has been introduced, I think the principle followed is to try the new one without an option, and falling back if it is detected that it's required. That would be much nicer than expecting the innocent user from understanding what `user-describe-topics-api` means.

Reply:
user-describe-topics-api is removed.